test/e2e/resource_check/resource_check_test.go

```diff
@@ -0,0 +1,72 @@
+package basic_test
+
+import (
+	"e2e-basic/common"
+	"os"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+// Check that there are no artefacts left over from
+// the previous 3rd party test.
+func resourceCheck() {
+
+	found, err := common.CheckNoTestPods()
+	if err != nil {
+		logf.Log.Error(err, "Failed to checking for test pods.")
+	} else {
+		Expect(found).To(BeFalse())
+	}
+
+	found, err = common.CheckNoPVCs()
```

Comment:
Nit: I assumed that if we were calling "CheckNoPVCs" that it would return true if there were no PVCs; because we are checking that there are no PVCs. Maybe the function name could be changed to something like "HasPVCs"?

Reply:
There are a bunch of functions prefixed with CheckNo, I will change all the prefixes to CheckFor